augur/application/cli/_multicommand.py

```diff
@@ -28,9 +28,7 @@ def get_command(self, ctx, name):
             module = importlib.import_module('.' + name, 'augur.application.cli')
             return module.cli
         except ModuleNotFoundError:
-            print(f"Error")
```

Comment:
We should probably print an 'invalid command' message to stderr when this happens

Reply:
The `click` library already prints this by default.

Here is an example when I put in `augur backen start`

Usage: augur [OPTIONS] COMMAND [ARGS]...
Try 'augur --help' for help.

Error: No such command 'backen'.

Reply:
I gotcha that makes sense.